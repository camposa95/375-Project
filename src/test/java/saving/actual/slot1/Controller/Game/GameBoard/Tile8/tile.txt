TerrainType: HILLS
DieNumber: 9
HasRobber: false
VertexIDs: [18, 19, 20, 29, 30, 31]
