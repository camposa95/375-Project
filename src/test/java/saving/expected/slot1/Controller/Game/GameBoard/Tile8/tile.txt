TerrainType: FORREST
DieNumber: 11
HasRobber: false
VertexIDs: [18, 19, 20, 29, 30, 31]
